content/en/docs/tasks/debug-application-cluster/debug-running-pod.md

```diff
@@ -0,0 +1,194 @@
+
+```shell
+kubectl exec cassandra -- cat /var/log/cassandra/system.log
+```
+
+You can run a shell that's connected to your terminal using the `-i` and `-t`
+arguments to `kubectl exec`, for example:
+
+```shell
+kubectl exec -it cassandra -- sh
+```
+
+For more details, see [Get a Shell to a Running Container](
+/docs/tasks/debug-application-cluster/get-shell-running-container/).
+
+## Debugging with Ephemeral Debug Container
+
+{{< feature-state state="alpha" for_k8s_version="v1.18" >}}
+
+{{< glossary_tooltip text="Ephemeral Containers" term_id="ephemeral-container" >}}
+are useful for interactive troubleshooting when `kubectl exec` is insufficient
+because a container has crashed or a container image doesn't include debugging
+utilities, such as with [distroless images](
+https://github.com/GoogleContainerTools/distroless). `kubectl` has an alpha
+command that can create ephemeral containers for debugging beginning with version
+`v1.18`.
+
+### Example debugging using Ephemeral Containers
+
+{{< note >}}
+The examples in this section require the `EphemeralContainers` [feature gate](
+/docs/reference/command-line-tools-reference/feature-gates/) enabled in your
+cluster and `kubectl` version v1.18 or later.
+{{< /note >}}
+
+You can use the `kubectl alpha debug` command to add ephemeral containers to a
+running Pod. First, create a pod for the example:
+
+```shell
+kubectl run ephemeral-demo --image=k8s.gcr.io/pause:3.1 --restart=Never
+```
+
+{{< note >}}
+This section use the `pause` container image in examples because it does not
+contain userland debugging utilities, but this method works with all container
+images.
+{{< /note >}}
+
+If you attempt to use `kubectl exec` to create a shell you will see an error
+because there is no shell in this container image.
+
+```shell
+kubectl exec -it pause -- sh
+```
+
+```
+OCI runtime exec failed: exec failed: container_linux.go:346: starting container process caused "exec: \"sh\": executable file not found in $PATH": unknown
+```
+
+You can instead add a debugging container using `kubectl alpha debug`. If you
+specify the `-i`/`--interactive` argument, `kubectl` will automatically attach
+to the console of the Ephemeral Container.
+
+```shell
+kubectl alpha debug -it ephemeral-demo --image=busybox --target=ephemeral-demo
+```
+
+```
+Defaulting debug container name to debugger-8xzrl.
+If you don't see a command prompt, try pressing enter.
+/ # ps auxww
+PID   USER     TIME  COMMAND
+    1 root      0:00 /pause
+   11 root      0:00 sh
+   16 root      0:00 ps auxww
+/ # ls -lh /proc/1/root/
+total 736K
+drwxr-xr-x    5 root     root         360 Feb 12 13:23 dev
+drwxr-xr-x    2 root     root        4.0K Feb 12 13:23 etc
+-rwxr-xr-x    1 root     root      725.1K Dec 20  2017 pause
+dr-xr-xr-x  167 root     root           0 Feb 12 13:23 proc
+dr-xr-xr-x   13 root     root           0 Feb 12 13:23 sys
+drwxr-xr-x    3 root     root        4.0K Feb 12 13:23 var
```

Comment:
The style guide does recommend separating commands (even examples) from their expected output. How about having just one sample command; for example:
```bash
# Run this inside the debug container
ps -p 1 -o command=
```
```
/pause
```

Reply:
I think the multiple textboxes are distracting to the description of `kubectl debug`, so if it's going to be a sticking point I'd rather just remove it entirely, which I've now done.